README.md

```diff
@@ -463,49 +463,73 @@ You can only build HybridApp if you have been granted access to [`Mobile-Expensi
     [url "https://github.com/"]
         insteadOf = ssh://git@github.com/
     ```
-3. The first time you build the app you will need to build YAPL (OldDot javascript logic). Simply run `npm run grunt:build:shared` from the `Mobile-Expensify` submodule
-    - The following runtime error often indicates that YAPL has not been built correctly: `undefined is not an object (evaluating'Store.ReportHistory.bindCacheClearingEvents')`
 
-At this point, the default behavior of some `npm` scripts will change to target HybridApp:
-- `npm run android` - build HybridApp for Android
-- `npm run ios` - build HybridApp for iOS
-- `npm run ipad` - build HybridApp for iPad
-- `npm run ipad-sm` - build HybridApp for small iPad
-- `npm run pod-install` - install pods for HybridApp
-- `npm run clean` - clean native code of HybridApp
+> [!Note]  
+> #### For external contributors
+>
+> If you'd like to modify the `Mobile-Expensify` source code, it is best that you create your own fork. Then, you can swap origin of the remote repository by executing this command:
+>
+> `cd Mobile-Expensify && git remote set-url origin <YOUR_FORK_URL>`
+>
+> This way, you'll attach the submodule to your fork repository.
+
+- Before building the app, you need to install dependencies. Run `npm install` from `Expensify/App` (this will also install dependencies for `Mobile-Expensify`)
```

Comment:
How about adding information how we detect if we should build `HybridApp` instead of standalone NewDot?

Reply:
I think it is enough to give reference to the script that decides if we use HybridApp scripts or standalone scripts